<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodedialog</class>
 <widget class="QDialog" name="nodedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>416</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Linguistic Variables</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listNodes">
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add variable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove variable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetMember">
       <property name="text">
        <string>Set membership function</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>Edit variable's name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>removeNode()</slot>
  <slot>addNode()</slot>
  <slot>editNode()</slot>
  <slot>selectAllNodes()</slot>
  <slot>selectNoneNodes()</slot>
 </slots>
</ui>
